src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+                description="We will help you find the right wallet based on the features you care about."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.futureTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/wallets/find-wallet/">
+                    List of wallets
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Crypto security 101"
+                description="Learn how to identify scams and how to avoid the most common tricks."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.dogeComputer)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/security/">Crypto security 101</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <Banner>
+              <BannerBody>
+                <h3>Things to consider when using Ethereum</h3>
+                <ul>
+                  <li>
+                    Each Ethereum transaction requires a fee in the form of ETH,
+                    even if you need to move different tokens built on Ethereum
+                    like the stablecoins USDC or DAI.
+                  </li>
+                  <li>
+                    Fees can be high depending on the demand so we recommend
+                    using <Link to="/layer-2/">Layer 2s</Link>.
+                  </li>
+                  <li>
+                    There are options for custodial and non-custodial wallets.
+                  </li>
+                </ul>
+              </BannerBody>
+              <BannerImage>
+                <GatsbyImage
+                  image={getImage(data.newRings)}
+                  alt=""
+                />
+              </BannerImage>
+            </Banner>
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/community/support/">
+                Ethereum and wallet support
+              </DocLink>
+              <DocLink to="/layer-2/">
+                Layer 2 - reducing transaction cost
+              </DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[2].id}>{tocItems[2].title}</h2>
+            <p>
+              Ethereum has led to the creation of new products and services that
+              can improve different areas of our lives. We're still in the early
+              stages but there's a lot to be excited about.
+            </p>
+            <CardGrid>
+              <Card
+                title="Decentralized Finance"
```

Comment:
Any reason we aren't using the acronyms here @konopkja? We should if we want to use the acronym in the button

<img width="344" alt="Screenshot 2022-07-07 at 16 06 28" src="https://user-images.githubusercontent.com/62268199/177807517-39938f7b-5550-4b23-a463-f7292f6e7844.png">


```suggestion
                title="Decentralized finance (DeFi)"
```

Reply:
Defi is a shortcut - as such it assumes some prior knowledge, thats who i did not wanted to use the acronym in the title, but thought in the button would be okay.

Reply:
I'm not sure I understand. We're using an acronym (DeFi) in the button without defining what it means. I'm suggesting we add it in parenthesis after the term is fully spelt out. Example: 

<img width="359" alt="Screenshot 2022-07-07 at 17 44 35" src="https://user-images.githubusercontent.com/62268199/177826887-ed6074b9-50d7-4126-a0ef-06da6e69cc14.png">